lib/data_migrator/offence_adder.rb

```diff
@@ -0,0 +1,106 @@
+# Convenience class for creating or destroying
+# an offence
+# Example: add a new scheme 11 offence
+#
+# fee_scheme_11 = FeeScheme.where(version: 11).where(name: 'AGFS').first
+# migrator = OffenceDataMigrator.new(
+#   fee_scheme: fee_scheme_11,
+#   offence_band: '17.1',
+#   description: 'Aiding, abetting, causing or permitting dangerous driving',
+#   contrary: 'Road Traffic Act 1988, s.2',
+#   year_chapter: '1988 c. 52'
+# )
+# migrator.up
+#
+require_relative 'offence_code_generator'
+
+module DataMigrator
+  class OffenceAdder
+    attr_reader :fee_scheme
+
+    def initialize(fee_scheme:, description:, offence_band:, contrary:, year_chapter:)
+      @fee_scheme = fee_scheme
+      @description = description
+      @offence_band = offence_band
+      @contrary = contrary
+      @year_chapter = year_chapter
+    end
+
+    def up
+      offence = find_or_create_offence!(attributes)
+      puts "-- found or created offence (#{offence.attributes})"
+    end
+
+    def down
```

Comment:
Why `up` and `down`? Is this a traditional thing. Why not `add` and `remove`?

Reply:
yeah, stand rails migration convention for schema changes